src/GeositeFramework/css/main.css

```diff
@@ -2120,3 +2120,95 @@ body.x-body {
 .arcgisSearch .searchGroup .searchInput:-moz-placeholder { /* Firefox 18- */
   color: #fff;
 }
+@media screen and (min-device-width: 737px) {
+    #single-plugin-toggle-footer {
+        display: none;
+    }
+}
+
+@media screen and (max-device-width: 736px) {
+    #toggle-plugin-container {
```

Comment:
Are the `!important` flags used in this block required? I thought that the rules defined within the media query would take precedent over other rules when the media query is active.

Reply:
Some of these do seem to be required, but I'll go through and check to make sure we keep only the minimal number of them.

Reply:
I pared these down to what seem to be the minimal required set.